androidsdk/src/main/java/com/wootric/androidsdk/Constants.java

```diff
@@ -30,4 +30,8 @@
     public static final long DAY_IN_MILLIS = 1000L *60L *60L *24L;
 
     public static final int NOT_SET = -1;
+
+    public static final long DEFAULT_RESURVEY_DAYS = 90L;
```

Comment:
@prabhatjha I checked the web beacon to be sure. On web beacon, if cookie `surveyed` exists, then it doesn't hit eligibility server. On code, that cookie's expiration date is supposed to be set with the `resurvey_throttle` value. However, that value is not being sent right now, which causes the cookie to expire when browsing session ends. Without that cookie, web beacon always hits eligibility server.